source/configuration_manual/mail_location/obox/other_s3-compatible_storages.rst

```diff
@@ -4,8 +4,12 @@
 Other S3-compatible Storages
 ==================================
 
-The configuration is likely to be similar to Amazon S3, but often the bucket
-name needs to be specified explicitly via the bucket parameter:
+This document covers the S3 compatible storage services. In case you want to
+use AWS S3 please refer to :ref:`amazon_s3`.
+
+The configuration for ``s3`` is likely to be similar to Amazon S3 ``aws-s3``,
+but often the bucket name needs to be specified explicitly via the bucket
```

Comment:
s/often the bucket name needs to/generally the bucket name must/

Reply:
yes good point!